docs/modules/ROOT/pages/configuration-store.adoc

```diff
@@ -0,0 +1,172 @@
+specific configurations. 
+
+NOTE: Multiple configuration properties files can be stored at either URI.
+
+Since it is common practice to define separate Helm chart values for each environment of a project's development lifecycle, it is 
+encouraged to define one shared base URI and respective environment specific URIs, each housing the relevant overrides.
+
+The following Helm chart configuration demonstrates the URI specifications for an example CI deployment:
+[source,yaml]
+----
+  env:
+    BASE_PROPERTY: <URI housing base property configurations>
+    ENVIRONMENT_PROPERTY: <URI housing CI-specific property overrides/augmentations>
+----
+
+==== Backend Storage Options
+The universal configuration store offers robust production ready configuration storage using https://developer.hashicorp.com/vault[Vault,role=external,window=_blank]. This is an excellent solution for large 
+enterprise applications. For smaller applications looking for a more light weight but still robust storage option, the configuration store 
+can be set to use https://github.com/TechnologyBrewery/krausening[Krausening,role=external,window=_blank]. The configuration store can also be set up to use in memory storage, this is best
+for quick local testing.
+
+The default property storage is Krausening. To change it to use one of the other options, set the environment
+variable `STORAGE_CLASS` to the desired value. The values currently available are `krausening`, `vault`, and `inMemory`.
+
+=== Usage
+After the configurations are ingested and reconciled within the store, the next step is to access them within a project using one of 
+the two available methods: the API endpoint or kubernetes resource injection. See the applicable sections below for more details on 
+utilizing these approaches.
+
+==== API Endpoint
+The configuration store includes a kubernetes service with the endpoints detailed below. The API endpoint provides a simple yet flexible way 
+to access any configurations from within a project.
+
+.Endpoints
+[cols="2,1,4"]
+|===
+| Endpoint Path | HTTP Method | Description
+
+| `/aissemble-properties/{groupName}/{propertyName}`
+| `GET`
+| Returns a JSON object containing the value of a property with the given group name and property name.
+
+| `/aissemble-properties/healthcheck`
+| `GET`
+| Returns a plain text string used for validating the service is reachable.
+
+|===
+
+Here is an example `curl` request to the configuration store service containing the properties files defined above:
+[source,bash]
+----
+curl http://configuration-store.config-store-ns.svc:8083/aissemble-properties/messaging/topic
+----
+
+Resulting JSON object:
+[source,json]
+----
+{
+  "groupName": "messaging",
+  "name": "topic",
+  "value": "ciTopic"
+} 
+----
+
+NOTE: The name of the configuration service may vary depending on the projects helm chart configuration, by default it is
+`http://configuration-store.config-store-ns.svc:8083`
+
+==== Resource Injection
```

Comment:
A: Should this be _Kubernetes_ Resource Injection to match what we called it above?